app/app.js

```diff
@@ -0,0 +1,54 @@
+(function(){
+	var app = angular.module('diplomacy', []);
+	
+	var server = 'http://diplomacy2.asilika.com:9494';
+	
+	app.controller('GamesController', ['$http', function($http){
+   
+		this.games = "Hello, these are the games";
+		var gamesCtrl = this;
+		
+		$http.get( server + '/api/rest/games').success(function(response){
+		  gamesCtrl.games = response.data.data;
+		});
+	}]);
+ 
+	app.controller('MatchesController', ['$http', function($http){
+  
+		this.matches = "Hello, these are the matches";
```

Comment:
Everywhere you're using `this` in the the controllers, you probably want to be using `$scope`